shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/ApiMapper.java

```diff
@@ -72,4 +79,44 @@ public interface ApiMapper {
      * @return update count
      */
     int updateByPrimaryKey(ApiDO record);
+
+    /**
+     * existed.
+     *
+     * @param id id
+     * @return existed
+     */
+    @Override
+    Boolean existed(@Param("id") Serializable id);
+
+    /**
+     * select api by query.
+     *
+     * @param query {@linkplain ApiQuery}
+     * @return {@linkplain List}
+     */
+    List<ApiVO> selectByQuery(ApiQuery query);
+
+    /**
+     * select api by ids.
+     * @param ids primary keys.
+     * @return {@linkplain ApiDO}
+     */
+    List<ApiDO> selectByIds(List<String> ids);
+
+    /**
+     * delete api.
+     *
+     * @param ids primary keys.
+     * @return rows int
+     */
+    int deleteByIds(List<String> ids);
+
+    /**
+     * select by primary key get VO.
+     *
+     * @param id primary key
+     * @return object by primary key VO
+     */
+    ApiVO selectByPrimaryKeyGetVO(@Param(value = "id") String id);
```

Comment:
remove `ApiVO selectByPrimaryKeyGetVO(@Param(value = "id") String id);`, assemble it in the `Service` layer.

Reply:
fixed it